doc/reference/reference_sql/sql_user_guide.rst

```diff
@@ -535,24 +547,25 @@ UUID support in SQL was added in Tarantool version 2.9.1.
 .. _sql_data_type_scalar:
 
 SCALAR can be used for
-:ref:`column definitions <sql_column_def_data_type>` but the individual column values have
-one of the preceding types -- BOOLEAN, INTEGER, DOUBLE, STRING, VARBINARY, or UUID.
-See more about SCALAR in the section
+:ref:`column definitions <sql_column_def_data_type>` and the individual column values have
+type SCALAR. See
 :ref:`Column definition -- the rules for the SCALAR data type <sql_column_def_scalar>`.
 The data-type may be followed by :ref:`[COLLATE collation-name] <sql_collate_clause>`.
-
+Prior to Tarantool version 2.10.1, individual column values had
+one of the preceding types -- BOOLEAN, INTEGER, DOUBLE, DECIMAL, STRING, VARBINARY, or UUID.
+Starting in Tarantool version 2.10.1, all values have type SCALAR.
 
 Any value of any data type may be NULL. Ordinarily NULL will be cast to the
 data type of any operand it is being compared to or to the data type of the
 column it is in. If the data type of NULL cannot be determined from context,
 it is BOOLEAN.
 
-All the SQL data types correspond to
+Most of the SQL data types correspond to
 :ref:`Tarantool/NoSQL types <details_about_index_field_types>` with the same name.
 There are also some Tarantool/NoSQL data types which have no corresponding SQL data types.
 If Tarantool/SQL reads a Tarantool/NoSQL value which has a type which has no SQL equivalent,
 Tarantool/SQL may treat it as NULL or INTEGER or VARBINARY.
-For example, ``SELECT "flags" FROM "_space";`` will return a column whose data type is ``'map'``.
+For example, ``SELECT "flags" FROM "_vspace";`` will return a column whose Lua type is ``'map'``.
```

Comment:
Thanks, and I did one more commit, I removed the word "Lua". Actually, as you know, this whole
paragraph will be obsolete because MAP support was recently added, but
we can handle that in a separate issue.